MarkBind/markbind
# Disable fuzzy link and include autolink feature in UG

**What is the purpose of this pull request?**




**Overview of changes:**
Partially address #1774
Add documentation to include example usage with URL and email, as well as some tricks to escape link conversion.
- fix one test

**Anything you'd like to highlight / discuss:**
- GitHub's similar autolink feature is documented [here](https://github.github.com/gfm/#autolinks-extension-)
- Seems like there is no proper way to disable this on GitHub, workarounds can be found [here](https://gist.github.com/alexpeattie/4729247)

**Testing instructions:**
Check PR preview UG section [here](https://deploy-preview-1779--markbind-master.netlify.app/userguide/formattingcontents#autolinks)


**Proposed commit message: (wrap lines at 72 characters)**
Disable fuzzy link and mention autolink feature in UG



---

**Checklist:** :ballot_box_with_check:
